grafana/archive/clean.sls

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
```

Comment:
This state only remove the directory containing extracted files, don't you want to also remove/uninstall Grafana itself?

Reply:
The `archive.extracted` state will extract all grafana software into `/opt/grafana-xxxxxx` directory. 
This clean state will remove that directory - because archive is simple tarball - removing the directory removes/uninstalls Grafana.